contributions/catalog/controllers/contribute.py

```diff
@@ -344,6 +470,48 @@ def create():
     return render_template('contribute/contribute.html', required_capabilities=required_capability_list,
                            post=json_contribute, user=session["name"],  token=session['oauth_token']['access_token'])
 
+# reviewers page
+@bp.route('/contributions/reviwers', methods=['GET'])
```

Comment:
```suggestion
@bp.route('/contributions/reviews', methods=['GET'])
```

Please rename this page to reviews. Thanks.

Reply:
changed